src/app/components/cards/MarkdownViewer.jsx

```diff
@@ -13,7 +14,14 @@ const remarkable = new Remarkable({
     breaks: true,
     linkify: false, // linkify is done locally
     typographer: false, // https://github.com/jonschlinkert/remarkable/issues/142#issuecomment-221546793
-    quotes: '“”‘’'
+    quotes: '“”‘’',
+    highlight: (src, lang) => {
+        if (lang && hljs.getLanguage(lang)) {
+            try {
+                return hljs.highlight(lang, src).value
+            } catch (e) {}
```

Comment:
error needs caught and handled properly

Reply:
What do you consider properly in this case? We fall back to not syntax highlighting if the method throws an error.